assets/src/dashboard/utils/validateGoogleAnalyticsIdFormat.js

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-const googleAnalyticsIdFormatRegex = /^ua-\d+-\d+$/;
+const gaTrackingFormatOrGa4MeasurementId = /^ua-\d+-\d+|g-\d{7}$/;
 
 export default function validateGoogleAnalyticsIdFormat(value = '') {
-  return Boolean(value.toLowerCase().match(googleAnalyticsIdFormatRegex));
+  return Boolean(value.toLowerCase().match(gaTrackingFormatOrGa4MeasurementId));
```

Comment:
is it more clear to have the "or" in code rather than in the regex? Otherwise, lgtm!

Reply:
you mean like `validateGoogleAnalyticsOrGA4MeasurementId` as opposed to `match(gaTrackingFormatOrGa4MeasurementId)`?  

i did the check in the regex so that it's only running the check once.